cmd/unused-exporter/exporter.go

```diff
@@ -133,11 +141,32 @@ func (e *exporter) Collect(ch chan<- prometheus.Metric) {
 					labels[k] = meta[k]
 				}
 				e.logger.Log("unused disk found", labels)
-				countByNamespace[meta["kubernetes.io/created-for/pvc/namespace"]] += 1
+				countByNamespace[meta.CreatedForNamespace()] += 1
 			}
 			for ns, c := range countByNamespace {
 				ch <- prometheus.MustNewConstMetric(e.count, prometheus.GaugeValue, float64(c), name, pid, ns)
 			}
+
+			for _, d := range disks {
+				m := d.Meta()
+
+				var ts float64
+				lastUsed := d.LastUsedAt()
+				if !lastUsed.IsZero() {
+					ts = float64(lastUsed.UnixMilli())
+				}
+
+				if m.CreatedForPV() == "" {
+					continue
+				}
+
+				ch <- prometheus.MustNewConstMetric(e.dlu, prometheus.GaugeValue, ts, name, pid,
+					d.ID(),
+					m.CreatedForPV(),
+					m.CreatedForPVC(),
+					m.Zone(),
+				)
+			}
```

Comment:
Any reason why this is done in a different loop than the one on line 133?

Reply:
Separation of concerns

Reply:
🤔 ok. I feel like it adds more repetition and obscures the code a little bit, but as said, I don't think it's blocking at the moment.

Reply:
I disagree; I think this enhances readability by keeping each loop for a single purpose; there's nothing wrong with a little repetition.

It's a stylistic choice and you're the repo owner so if you want it changed I will change it.

Reply:
Nah, as said, it isn't a big of a deal. I'll leave it to @Pokom for a last review, but I'm happy to merge this.

Reply:
It looks good to me! I don't have a strong opinion one way or another on it, and the additional tests helped me better understand the coalesce method 🙇🏻

Reply:
Thanks for the review, fellas 👍 
Can we merge this?